buffalo/algo/bpr.py

```diff
@@ -108,7 +108,7 @@ def _get_topk_recommendation(self, rows, topk, pool=None):
         return zip(rows, topks)
 
     def _get_most_similar_item(self, col, topk, pool):
-        return super()._get_most_similar_item(col, topk, self.Q, self.opt._nrz_Q, pool)
+        return self._get_most_similar(col, topk, self.Q, self.opt._nrz_Q, pool)
```

Comment:
why is `_get_most_similar_user` only implemented in als, and not in bpr, w2v, cfr?

Reply:
cause I have little knowledge about those algorithms.
calling ```_get_most_similar_user``` for those algos will raise ```NotImplementedError```.
I hope someone who knows better to implement it.

Reply:
ok